Cannot find module 'jsbi' when using ion-js >= 3.1.0

Code that depends on ion-js >= 3.1.0 produces the following error:
```
Error: Cannot find module 'jsbi'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (ionjstest/node_modules/ion-js/dist/commonjs/es6/IonParserBinaryRaw.js:13:32)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3
```

I suspect this is because jsbi is declared in package.json's `devDependencies` list, rather than [regular] `dependencies`.

--- a/README.md
+++ b/README.md
@@ -18,7 +18,7 @@ You can use this library either as a Node.js module or inside an HTML page.
 
 1. Add `ion-js` to your dependencies using `npm`
     ```
-    npm install --save-dev ion-js
+    npm install --save ion-js
     ```
 1. Use the library to read/write ion data. Here is an example that reads Ion data from a Javascript string
     ```javascript

--- a/package.json
+++ b/package.json
@@ -71,11 +71,13 @@
     "mocha": "^6.2.0",
     "mocha-typescript": "^1.1.17",
     "nyc": "^14.1.1",
-    "jsbi": "^3.1.1",
     "semantic-release": "^15.13.24",
     "source-map-support": "^0.5.13",
     "ts-node": "^8.4.1",
     "typedoc": "^0.15.0",
     "typescript": "^3.6.2"
+  },
+  "dependencies": {
+    "jsbi": "^3.1.1"
   }
 }

